src/sparseml/pytorch/image_classification/utils/helpers.py

```diff
@@ -428,6 +257,8 @@ def save_model_training(
     :param epoch: integer representing umber of epochs to
```

Comment:
There are two variables present in docstr, which are not present in the function parameters: input_shape and conver_quat. Could you fix it?

Reply:
Good catch! thanks.

Reply:
Fixed! thanks